recipes/glew/all/conanfile.py

```diff
@@ -0,0 +1,119 @@
+import os
+import glob
+from conans import ConanFile, CMake, tools
+
+class GlewConan(ConanFile):
+    name = "glew"
+    description = "The GLEW library"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://github.com/nigels-com/glew"
+    topics = ("conan", "glew", "opengl", "wrangler", "loader", "binding")
+    license = "MIT"
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    generators = "cmake"
+    settings = "os", "arch", "build_type", "compiler"
+    options = {"shared": [True, False]}
+    default_options = {"shared": False}
+    _source_subfolder = "source_subfolder"
+
+    def requirements(self):
+        self.requires("opengl/system")
+        self.requires("glu/system")
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("%s-%s" % (self.name, self.version), self._source_subfolder)
+        if "patches" in self.conan_data:
+            for patch in self.conan_data["patches"][self.version]:
+                tools.patch(**patch)
+#        tools.replace_in_file("%s/build/cmake/CMakeLists.txt" % self._source_subfolder, "include(GNUInstallDirs)",
+#"""
+#include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+#conan_basic_setup()
+#include(GNUInstallDirs)
+#""")
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+        cmake.definitions["BUILD_UTILS"] = "OFF"
+        cmake.definitions["CONAN_GLEW_DEFINITIONS"] = ";".join(self._glew_defines)
```

Comment:
is this actually used anywhere?

Reply:
it used to be in https://github.com/bincrafters/conan-glew/blob/581c58940d8c6d09abc74d6f4dcb1ef41cbc8902/FindGLEW.cmake.in#L21, but that won't make it in CCI !